monkey/monkey_island/cc/ui/src/components/report-components/security/ScannedServers.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactTable from 'react-table'
+import Pluralize from 'pluralize'
```

Comment:
we should probably add `;` to the end or remove it from `import React from 'react';`

Reply:
fixed